languages/elixir/config.json

```
@@ -0,0 +1,15 @@
+{
+  "language": "Elixir",
+  "version": 3,
+  "active": true,
+  "blurb": "Elixir is a dynamic, functional language designed for building scalable and maintainable applications. Elixir leverages the Erlang VM, known for running low-latency, distributed and fault-tolerant systems, while also being successfully used in web development and the embedded software domain.",
+  "online_editor": {
+    "indent_style": "space",
+    "indent_size": 2
+  },
+  "exercises": {
+    "concept": [],
+    "practice": []
+  },
+  "foregone": []
```

```suggestion
  "foregone": ["robot-name"]
```

based on https://github.com/exercism/elixir/blob/master/config.json#L6

---

Ah, missed that.  Thanks!